Userland/Libraries/LibJS/Runtime/RegExpConstructor.cpp

```diff
@@ -27,6 +38,83 @@ void RegExpConstructor::initialize(Realm& realm)
     define_native_accessor(realm, *vm.well_known_symbol_species(), symbol_species_getter, {}, Attribute::Configurable);
 
     define_direct_property(vm.names.length, Value(2), Attribute::Configurable);
+
+    // Additional properties of the RegExp constructor, https://github.com/tc39/proposal-regexp-legacy-features#additional-properties-of-the-regexp-constructor
+    define_native_accessor(realm, "input", input_getter, {}, Attribute::Configurable);
```

Comment:
This also needs a setter, see https://github.com/tc39/proposal-regexp-legacy-features#set-regexpinput--val